packages/cli/package.json

```diff
@@ -17,8 +17,8 @@
     "stackgen": "npx tsx stackgen",
     "yalc": "npx yalc publish",
     "compile": "tsup --dts --out-dir ./dist ./index.ts",
-    "clean": "find . -name \"*.js\" -not -path \"./node_modules/*\" -delete && find . -name \"*.d.ts\" -not -path \"./node_modules/*\" -delete",
-    "lint": "eslint --ext .ts src"
+    "lint": "eslint --ext .ts src",
+    "clean": "rimraf ./dist"
```

Comment:
Does it matter if it's `./dist` or `./dist/` or `./dist/*`?

Reply:
Tested it, works fine.